web/frontend/src/components/camera/camera.vue

```diff
@@ -65,10 +65,19 @@ const viewCamera = async (isOn: boolean) => {
 
     manageStreamStates();
 
-    if (camerasOn === 1) {
+    /*
+     * Add the stream if 1 or more camera are present
+     * The stream can exist on base, camera, or both.
+     */
+    if (camerasOn > 1) {
```

Comment:
Yea this isn't great as we'll get the stream already open error. We should only call this if the stream isn't already active.

Reply:
Agreed. Its possible that a better fix is to update the SDK. I'm not entirely sure why we aren't receiving a `track` event for the camera card but are for the base card. But feels like either:
- Edge case in SDK where the event is not triggering in this case.
- Move the StreamClient up in the component tree so that only a single client is managing streams. Currently we can have multiple clients per actual camera.

But this might need to be a "bandaid fix" until something better can be done.